minard/views.py

```diff
@@ -1296,6 +1318,33 @@ def pca_run_detail(run_number):
                            run_number=run_number,
                            run=run)
 
+@app.route('/burst_run_detail/<run_number><subrun><sub>')
+def burst_run_detail(run_number, subrun, sub):
+    server = couchdb.Server("https://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
+    db = server["burst"]
+
+    for row in db.view('_design/burst/_view/burst_by_run'):
+        if (int(row.key[0]) == int(run_number)):
+            if (int(row.key[1]) == int(subrun)):
+                if (int(row.key[2]) == int(sub)):
```

Comment:
Could this be simplified to:

```python
if row.key[0] == run and row.key[1] == subrun and row.key[2] == sub:
```
Also, are the explicit int conversions on the row necessary here?